django/db/backends/base/operations.py

```diff
@@ -666,9 +666,9 @@ def explain_query_prefix(self, format=None, **options):
         if format:
             supported_formats = self.connection.features.supported_explain_formats
             if format.upper() not in supported_formats:
-                msg = '{0} is not a recognised format. '.format(format.upper())
+                msg = '{0} is not a recognised format.'.format(format.upper())
                 if supported_formats:
-                    msg += 'Allowed formats: {0}'.format(', '.join(supported_formats))
+                    msg +=  'Allowed formats: {0}'.format(', '.join(supported_formats))
```

Comment:
Oops. I think you meant to put the space to the _**right**_ of the quote!

Reply:
Hah, my bad! Thanks for spotting